src/homework7/2/tests.py

```diff
@@ -0,0 +1,36 @@
+from runner import runner
+import unittest
+
+
+class TestRunner(unittest.TestCase):
+    def test_run(self):
```

Comment:
too many test cases in one test. please split them (you can use ddt lib here to make it more simple)